orion/sites/admin/src/programs/programEdit.js

```diff
@@ -178,9 +178,43 @@ export class ProgramEditPage extends React.Component {
 }
 
 function checkProgram(program) {
-  // *todo* change this later!
+  let checkpro=true;
+
+  let programId = program.programId
+
+  //checks if programId contains special characters
+  var iChars = "~`!#$%^&*+=-[]\\\';,/{}|\":<>?";
```

Comment:
could we get a better named variable to explain it more?
maybe something like `invalidChars`?

Also `let` is better to use than `var`. It's a javascript scope reason. If you're not careful with `var`, the variable can live on longer than you'd expect. In other words, when this function ends, var may still be alive and can be used elsewhere! Javascript is weird!